projects/cyclegan_reference_simulations/README.md

```diff
@@ -0,0 +1,132 @@
+by the climate-specific amount; one year is given for each of these simulations
+to spin up.  Initial conditions for the C384 simulations are derived from
+restart files from the first of each post-spin-up year of the C48 simulations in
+each climate.  These initial conditions were derived using the `chgres_cube`
+tool of the `UFS_UTILS` project using [the argo workflow defined in the fv3net
+repository](https://github.com/ai2cm/fv3net/tree/master/workflows/argo#restart-files-to-nggps-initial-condition-workflow).
+Note that for two select cases (year 2023 in the 0 K climate and year 2020 in
+the +4 K climate) we needed to restart from restart files one month prior to
+avoid a crash upon initialization.  The initialization crashes can be traced
+back to the fact that sharp topographic features that exist at C384 resolution
+do not exist at C48 resolution, and so sometimes the state interpolated to the
+finer grid is not initially compatible with the fine resolution topography.
+
+The initial conditions for the ramped simulations were derived from C48 restart
+files from January 1st, 2017.  The C48 simulation was started directly using
+these restart files, and the C384 simulation was started from an initial
+condition constructed with the `chgres_cube` tool.
+
+The C48 2016-01-01 00Z initial condition can be found in our cloud storage here:
+```
+gs://vcm-ml-raw/2020-11-05-GFS-month-start-initial-conditions-year-2016/2016010100
+```
+and can be made available upon request.
+
+## Ramped sea surface temperature forcing data
+
+The ramped simulations require the definition of a netCDF file containing the
+sea surface temperature on a latitude-longitude grid defined at monthly time
+intervals.  We derived this forcing by loading in the climatological sea surface
+temperature, defined for each month of the year, and repeating it for 51 months
+(four years and three months).  We added a linearly increasing perturbation to
+this time series starting at the start of the fourth month.  The evolution of
+this perturbation can be found in the graph below.
+
+![sst-offset-timeseries.png](sst-offset-timeseries.png?raw=true)
+
+A script, `generate_ramped_sst.py`, that generates this ramped SST forcing file
+is also included for reference.  See also
+[here](https://github.com/ai2cm/fv3gfs-fortran/blob/cb106f2eb806e8c635d28d8b76ee8e80a0e20bc3/tests/pytest/prescribed_ssts.py#L3-L28)
+for more information on how to generate an SST forcing file that can loaded by
+FMS.
+
+## Standard forcing data
+
+We run FV3GFS using standard forcing data provided for the UFS weather model.
+This consists of:
+
+- Grid data: `gs://vcm-fv3config/data/grid_data/v1.0`
+- Orographic forcing data: `gs://vcm-fv3config/data/orographic_data/v1.0`
+- Surface forcing data: `gs://vcm-fv3config/data/base_forcing/v1.1`
+
+Public versions of this data can be found
+[here](https://www.nco.ncep.noaa.gov/pmb/codes/nwprod/gfs.v16.3.7/fix/), and the
```

Comment:
Looking at this URL, it's unclear what the relationship is between these files and the `gs://` data. Not sure if there's any context to add there.